google/cloud/bigtable/internal/async_bulk_apply_test.cc

```diff
@@ -288,6 +285,179 @@ TEST_F(AsyncBulkApplyTest, AsyncBulkApplyFailureRetry) {
   EXPECT_TRUE(cq_impl_->empty());
 }
 
+TEST_F(AsyncBulkApplyTest, TooManyFailures) {
+  bigtable::BulkMutation mut{
+      bigtable::SingleRowMutation("foo2",
+                                  {bigtable::SetCell("f", "c", 0_ms, "v2")}),
+      bigtable::SingleRowMutation("foo3",
+                                  {bigtable::SetCell("f", "c", 0_ms, "v3")}),
+  };
+
+  // We give up on the 5th error.
+  const int limited_error_count = 4;
```

Comment:
Consider instead:

```
auto constexpr kErrorCount = 4;
```

Also: is there a benefit from testing with 4 errors vs. 1 or 2 or 3? Pick the smallest number that would do the trick.

Reply:
Done.